openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
     self.hostname = socket.gethostname()
     self.stations = collections.defaultdict(Station)
```

Comment:
now that i look at it more, i actually think it might be preferable to include `station_id` in the initial ping response (instead of requiring a second RTT to get it), and indexing this based on `station_id` instead of `(host, port)`.  host and port can then be stored as attributes on the Station tuple.  And you could do something like:
```
class Station(namedtuple(...)):
  @property
  def hostport(self):
    return self.host, self.port
```
And then when you need to do socket things you can pass in `my_station.hostport`.